### ISSUE TITLE
XPlane2Blender 3.5.0 beta 2 & double sided property.

### ISSUE THREAD

**Issue by Bridgemaster** [der-On/XPlane2Blender#378]
When exporting from Blender 2.79 with XPlane2Blender 3.5.0b2 objects lose their double sided properties.
Same problem found with 3.4.0 .
Original file was built with 2.69 and sucessfully exported with XPlane2Blender 3.20.14.

Link to .obj and .blend files
https://1drv.ms/f/s!AuHeisCReA02iWN7-Sa7csVqTogS

**Comment by tngreene** [comment]
Firstly let me say that that is a beautiful bridge! I always love it when people post their .blend files because they're always amazing looking. I am hoping very much that we can find a way to preserve the work you've done!

Sadly for right now, all I can say is that double sided properties as a feature are well and truely gone. I've asked the Laminar Research art team how they might model something like this without double sided properties and I'll let you know when they have a response back. In the mean time I encourage you to ask other experienced scenery modelers in X-Plane forms (which we do not own or endorse) like [x-plane.org's](https://forums.x-plane.org/index.php?/forums/forum/45-3d-modeling/) 3D modeling forum.

Someone might provide some good tips there.

**Comment by airfightergr** [comment]
@Bridgemaster you could model those faces in 3D. The amount of extra geometry is not that huge to create noticeable performance penalties.

Otherwise, you can also use LOD, to show those details only when the viewer is in close distance. You can't pick up such details 10 miles out, or use rectangle faces with a texture.

**Comment by Bridgemaster** [comment]
It’s not a problem, I also have Blender 2.69 on my PC, exporting with XPlane2Blender 3.2 so can use that when needs be. I was just curious to know where the feature had gone with the new exporter, now I’m aware of the reason I can work around it if I use Blender 2.79 in future. Thanks again for the clarification.

 

I would be interested to hear what the art team might say as I’m an admirer of their landmark objects especially the London ones and the texturing on the Gherkin building, ones which I had made myself some time ago. I wonder if they are also going to do the Mayor’s office, City Hall, now that’s a challenge and I speak from experience,

**Comment by Bridgemaster** [comment]
Thanks for your interest, Ted Green gave me a link illustrating the reasons for deprecation and whilst I found most of the explanation hard to follow it was a useful article. I shall certainly look for other methods in future.

**Comment by Bridgemaster** [comment]
Thank you Gentlemen, appreciated your input very much. Definitely not an issue for me anymore now I know the logic behind it.
